setup.cfg

```diff
@@ -0,0 +1,2 @@
+[bdist_wheel]
+universal = 1
```

Comment:
This declares that your module is pure python and will work on both 2 and 3.   
When released as a wheel this tells packaging tools about your compatibility even before they download the module.